Automattic/wp-calypso
# Use new image file to fix blur and cropping

Related to https://github.com/Automattic/wp-calypso/issues/74014

## Proposed Changes

* Update Built By WP image to remove blur and cropping on the side of the circle.

Before | After
--|--
![built-by-before](https://user-images.githubusercontent.com/140841/222781644-99e2b0ca-d577-4e91-aa48-47b7d17f6e28.png)  | ![built-by-after](https://user-images.githubusercontent.com/140841/222781656-c20c75b5-004e-42c6-ac3f-2073eda81857.png)

## Testing Instructions



* Load the PR and view /marketing/tools/:site in Safari
* The Built By WP logo should look sharp and clean
* View in Chrome and Firefox. The image should look good in all cases.

## Pre-merge Checklist
